docs/src/jsx/DuckScene.js

```diff
@@ -0,0 +1,39 @@
+//  Copyright (c) 2019-present, GM Cruise LLC
```

Comment:
Wonder if we should update existing file headers to be `2019` as well?

Reply:
Haha oh no how is this going to work with our linter that checks for one consistent header (at least in webviz-core)! 😮